### DISCUSSION TITLE
how to hide this in public?

### DISCUSSION THREAD

**Discussion by Shahzadqayyum** [YOURLS/YOURLS#3249] [General]
Connected to database blogscha_short on localhost 
SQL perform: SELECT option_name, option_value FROM url_options WHERE 1=1 (0.00030 s)
cURL error 60: SSL certificate problem: certificate has expired (POST on https://api.yourls.org/core/version/1.0/)
SQL perform: UPDATE url_options SET option_value = 'O:8:"stdClass":4:{s:15:"failed_attempts";i:24;s:12:"last_attempt";i:1647189852;s:11:"last_result";s:0:"";s:15:"version_checked";s:5:"1.8.2";}' WHERE option_name = 'core_version_checks' (0.00026 s)
Check for new version: no

its showing both private and public after footer.

**Comment by ozh** [comment]
This appeared after you added `define ('YOURLS_DEBUG', true);` to your `config.php`
